plugins/package-managers/pub/src/main/kotlin/Pub.kt

```diff
@@ -172,6 +172,14 @@
         val archive = when {
             Os.isWindows -> "windows/flutter_windows_$flutterVersion.zip"
             Os.isLinux -> "linux/flutter_linux_$flutterVersion.tar.xz"
+            Os.isMac -> {
+                val arch = when(val thisDeviceArch = System.getProperty("os.arch")) {
```

Comment:
I Have Added Space After when

Reply:
Please squash those commits. In this project we prefer rewriting the respective commit instead of adding a new commit to address review comment. You can just force-push to the existing PR instead of creating a new one. Please also see https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md.